diff --git a/CHANGELOG.txt b/CHANGELOG.txt
index d02f4d4..415e231 100644
--- a/CHANGELOG.txt
+++ b/CHANGELOG.txt
@@ -1,6 +1,7 @@
 File (Field) Paths 7.x-1.x-dev, xxxx-xx-xx (development release)
 --------------------------------------------------------------------------------
 
+- #1714596: Fixed issue with file_attach_update() introduced in Drupal 7.15.
 - #1429238 by ocanzillon: Fixed Drush command syntax error.
 - #1334448 by pianodavid, Deciphered: Fixed issue with revision flag.
 - Changed a large chunk of core functionality to simplify.
diff --git a/filefield_paths.module b/filefield_paths.module
index eb2e990..62b5ded 100644
--- a/filefield_paths.module
+++ b/filefield_paths.module
@@ -257,6 +257,7 @@ function filefield_paths_entity_update($entity, $type) {
       $entity->revision = FALSE;
     }
     // Save any changes back to the database.
+    $entity->original = isset($entity->original) ? $entity->original : NULL;
     field_attach_update($type, $entity);
     if (isset($entity->revision)) {
       // Restore revision flag so that other modules can process it if needed.
