diff --git modules/content_migrate/content_migrate.module modules/content_migrate/content_migrate.module
index 35c11be..fb00aee 100644
--- modules/content_migrate/content_migrate.module
+++ modules/content_migrate/content_migrate.module
@@ -423,7 +423,7 @@ function content_migrate_content_migrate_field_alter(&$field_value, $instance_va
       $field_value['module'] = 'image';
       $field_value['type'] = 'image';
       // default_image is now a field setting.
-      $field_value['settings']['default_image'] = $instance_value['widget']['settings']['default_image'];
+      $field_value['settings']['default_image'] = empty($instance_value['widget']['settings']['default_image'])?0:$instance_value['widget']['settings']['default_image'];
       break;
   }
 }
